nixos/modules/services/misc/jellyfin.nix

```diff
@@ -29,6 +29,14 @@ in
         default = "jellyfin";
         description = "Group under which jellyfin runs.";
       };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Open ports in the firewall for the media server.
```

Comment:
Can you document the fact that it will open the default ports, and if changed in the webui, this option will not entirely work?

Reply:
I didn't realise you could set them there.  Updated.  Thanks